int16 drone_id
int16 round_id
sensor_msgs/Image image
std_msgs/Header header
std_msgs/Float32MultiArray data
std_msgs/Float32MultiArray shift_matrix
std_msgs/Float32MultiArray trans_matrix